utils/swift_build_support/swift_build_support/products/swiftformat.py

```diff
@@ -0,0 +1,101 @@
+# swift_build_support/products/swiftformat.py -----------------*- python -*-
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https://swift.org/LICENSE.txt for license information
+# See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+# ----------------------------------------------------------------------------
+
+import os
+import platform
+
+from build_swift.build_swift.constants import MULTIROOT_DATA_FILE_PATH
+
+from . import cmark
+from . import foundation
+from . import libcxx
+from . import libdispatch
+from . import libicu
+from . import llbuild
+from . import llvm
+from . import product
+from . import swift
+from . import swiftpm
+from . import swiftsyntax
+from . import xctest
+from .. import shell
+
+
+class SwiftFormat(product.Product):
+    @classmethod
+    def product_source_name(cls):
+        """product_source_name() -> str
+
+        The name of the source code directory of this product.
+        """
+        return "swift-format"
+
+    @classmethod
+    def is_build_script_impl_product(cls):
+        return False
+
+    @classmethod
+    def is_swiftpm_unified_build_product(cls):
+        return True
+
+    def run_build_script_helper(self, action, additional_params=[]):
+        script_path = os.path.join(
+            self.source_dir, 'build-script-helper.py')
+
+        configuration = 'release' if self.is_release() else 'debug'
+
+        helper_cmd = [
+            script_path,
+            action,
+            '--toolchain', self.install_toolchain_path(),
+            '--config', configuration,
```

Comment:
I think you named this parameter `--configuration`, not `--config` in `build-script-helper.py` of `swift-format`. I am slightly inclined to change the parameter definition in the `swift-format` so it’s the same as for the stress tester, but don’t have strong opinions on it.

Reply:
Good catch, but it should be the other way: we should update the stress tester to use `--configuration` to match the swift-build option name.

Reply:
Thanks for catching this. I updated the name here.